src/interfaces/alteration-interfaces/alteration-interface.ts

```diff
@@ -1,6 +1,6 @@
 import {
   BusinessContactIF,
-  BusinessInformationIF,
+  BusinessInformationIF, NameRequestApplicantIF,
```

Comment:
Looks like `NameRequestApplicantIF` isn't used here.

Reply:
thanks! Will remove if obsolete

Reply:
removed in next commit!